examples/datasource-streaming-websocket/streaming-websocket-plugin/src/QueryEditor.tsx

```diff
@@ -28,21 +26,12 @@ export class QueryEditor extends PureComponent<Props> {
 
     return (
       <HorizontalGroup>
-        <FormField
-          width={4}
-          value={constant}
-          onChange={this.onConstantChange}
-          label="Constant"
-          type="number"
-          step="0.1"
-        />
-        <FormField
-          labelWidth={8}
-          value={queryText || ''}
-          onChange={this.onQueryTextChange}
-          label="Query Text"
-          tooltip="Not used yet"
-        />
+        <InlineField label="Constant" labelWidth={16}>
```

Comment:
Same as ^ for these fields. 

Also while at it, can you change from `HorizontalGroup` to `Stack`? Think horizontal group is about to be phased out.